cli/tests/integrations/test_config.py

```diff
@@ -66,12 +66,15 @@ def test_get_missing_property(env):
     _get_missing_value('missing.property', env)
 
 
-def test_invalid_dcos_url(env):
-    stderr = b'Please check url \'abc.com\'. Missing http(s)://\n'
-    assert_command(['dcos', 'config', 'set', 'core.dcos_url', 'abc.com'],
-                   stderr=stderr,
-                   returncode=1,
+def test_dcos_url_without_scheme(env):
+    old = b'http://dcos.snakeoil.mesosphere.com'
+    new = b"abc.com"
+    out = b"[core.dcos_url]: changed from '%s' to 'https://%s'\n" % (old, new)
```

Comment:
python3 fun!
https://docs.python.org/3.0/whatsnew/3.0.html#text-vs-data-instead-of-unicode-vs-8-bit